client/webserver/site/src/js/locales.ts

```diff
@@ -94,6 +94,8 @@ export const ID_MATCH_STATUS_REFUND_PENDING = 'MATCH_STATUS_REFUND_PENDING'
 export const ID_MATCH_STATUS_REDEEM_PENDING = 'MATCH_STATUS_REDEEM_PENDING'
 export const ID_MATCH_STATUS_COMPLETE = 'MATCH_STATUS_COMPLETE'
 export const ID_TAKER_FOUND_MAKER_REDEMPTION = 'TAKER_FOUND_MAKER_REDEMPTION'
+export const ID_REFUND_COMING_YOUR_WAY = 'REFUND_COMING_YOUR_WAY'
```

Comment:
Let's go with `REFUND_IMMINENT` so in the future it's clear that this is the stage where lockTime is passed and it's gonna happen any minute.  It's a bit colloquial for the code IMO.  The other one (`REFUND_WILL_HAPPEN_AFTER`) is clearly during the lockTime countdown.

Reply:
Sure, applied.